From: Simon McVittie <smcv@debian.org>
Date: Sun, 17 Feb 2019 14:18:48 +0000
Subject: Re: Bug#922516: missing binary gtk-query-immodules-3.0

Please add /usr/lib/x86_64-linux-gnu/libgtk-3-0 to the PATH when running
tests that rely on gtk-query-immodules-3.0, or invoke it by its full
path. Note that this is a Debian-specific change and should not be
upstreamed.

Forwarded: not-needed
Last-Update: 2019-12-27
---
 src/tests/runtest | 11 +++++++----
 1 file changed, 7 insertions(+), 4 deletions(-)

diff --git a/src/tests/runtest b/src/tests/runtest
index 1fcc928..71fc993 100755
--- a/src/tests/runtest
+++ b/src/tests/runtest
@@ -37,12 +37,15 @@ ibus-keypress
 test-stress
 "
 IBUS_SCHEMA_FILE='org.freedesktop.ibus.gschema.xml'
-GTK_QUERY_MODULE=gtk-query-immodules-3.0-32
+#GTK_QUERY_MODULE=gtk-query-immodules-3.0-32
+# On Debian with multiarch support, gtk-query-immodules-3.0 is under 
+# /usr/lib/x86_64-linux-gnu/libgtk-3-0/
+GTK_QUERY_MODULE=/usr/lib/`dpkg-architecture -qDEB_HOST_MULTIARCH`/libgtk-3-0/gtk-query-immodules-3.0
 MACHINE=`uname -m`
 
-if test x"$MACHINE" = xx86_64 ; then
-    GTK_QUERY_MODULE=gtk-query-immodules-3.0-64
-fi
+###if test x"$MACHINE" = xx86_64 ; then
+###    GTK_QUERY_MODULE=gtk-query-immodules-3.0-64
+###fi
 
 retval=0
 
